docs/variants/talos_2/hardware-matrix.md

```diff
@@ -16,6 +16,7 @@ coreboot port on the Talos II platform.
 | **Network**                    | Local network wired connection                                |
 | **Attached devices**           | 1. SanDisk USB 3.2Gen1 16 GB                                  |
 |                                | 2. Logitech, Inc. Keyboard K120                               |
+|                                | 3. Micron 1100 256GB M.2 SATA3                                |
```

Comment:
The `attached devices` list should be split as for any other Dasharo platforms. Take a look for an example this one: https://docs.dasharo.com/variants/protectli_vp46xx/hardware-matrix/

Reply:
Try pulling those changes and opening the preview of the `.md` file. In my case list is nicely split into rows.

Reply:
I've found a different mistake. SSD in other docs have their own category

Reply:
List should be splited in meanings: another category for SSDs, another for attached USB devices, etc.